services/fuel-core/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # v0.10.1 +plus
-FROM ghcr.io/fuellabs/fuel-core:sha-7b34fa8
+FROM ghcr.io/fuellabs/fuel-core:v0.11.0
```

Comment:
@camsjams Shouldn't it be `v0.11.2`?
- https://github.com/FuelLabs/fuel-core/releases